docs/variants/qemu_q35/building-manual.md

```diff
@@ -0,0 +1,48 @@
+#Dasharo (UEFI) v0.1.0 for QEMU Q35 - Building Manual
+
+In order to build the `Dasharo (UEFI) OVMF firmware image for QEMU Q35`, please follow the given steps to build the Dasharo firmware image from the EDK2 repository:
```

Comment:
Please wrap all markdown documents (except markdown syntax) at 80 characters. Remember to do it every time from now on.

Reply:
This is fixed,